### DISCUSSION TITLE
Add the ability to rotate images in 90-degree increments

### DISCUSSION THREAD

**Discussion by clareclever** [nextcloud/all-in-one#2227] [Ideas] [answered]
I have the AIO docker image and am wanting to upload some photos, some of which need to be rotated. Unless I am missing something it seems like the only way to rotate images is through the scroll wheel that doesn't allow you to rotate the image in common increments. Am I missing something or is that something that could be added?

**Comment by szaimen** [comment, marked as answer]
Hi, I guess this should rather be a feature request for nextcloud/server or nextcloud/android or nextcloud/ios. Feel free to open a feature request there.

**Reply by clareclever** [reply]
@szaimen, Thanks for the reply. I don't really understand how all this works. This project uses the code from the main Nextcloud/Server? Then the IOS app is an off shoot of that to access a Nextcloud instance?

**Reply by szaimen** [reply]
> This project uses the code from the main Nextcloud/Server? Then the IOS app is an off shoot of that to access a Nextcloud instance?

Yes, correct
